core/ulog/README.md

```diff
@@ -0,0 +1,44 @@
+# Logging package
+
+ulog provides an API wrapper around the logging library (zap Logger)
+The logger is instantiated as logger with default options and can be configured
+via `Configure()` API and provided YAML configuration.
+
+```go
+package main
+
+import "go.uber.org/fx/core/ulog"
+
+func main() {
+  // Initialize logger object
+  log := ulog.Logger()
+
+  // Optional, configure logger with configuration preferred by your service
+  logConfig := ulog.Configuration{}
+  log.Configure(&logConfig)
+
+  // Use logger in your service
+  log.Info("Message describing loggging reason", "key", "value")
+}
+```
+
+Note that the log methods (`Info`,`Warn`, `Debug`) takes parameter as key value pairs (message, (key, value)...)
+
+ulog configuration can be defined in multiple ways, either by writing the struct yourself, or describing in the YAML
+and populating using config package.
+
+* Defining config structure:
+
+```go
+loggingConfig := ulog.Configuration{
+  Stdout: true,
+}
+```
+
+* Configuration defined in YAML:
+
+```yaml
+  logging:
```

Comment:
nit: de-dent these two the beginning of the line (and the next two lines 2 spaces in as well)